### DISCUSSION TITLE
New badges archivement appear!

### DISCUSSION THREAD

**Discussion by thuongtruong1009** [community/community#20443] [Other Feature Feedback, Questions, & Ideas]
Hi guys!
I just got 2 badges from Github today. They are Starstruck and Quickdraw:
* Starstruck: appear when you create a repository and get many stars.
* Quickdraw: appear when you create a new PR and they are lost within 5 minutes of opening
![image](https://user-images.githubusercontent.com/71834167/178769787-0f60243f-1c2f-4aea-a886-eb001d264b55.png)

You can consult and admire them from my GitHub profile at [thuongtruong1009](https://github.com/thuongtruong1009)
Have a nice day! 🥰

**Comment by LuigiImVector** [comment]
There is also Pair Extraordinaire for co-authoring commits on merged PR

![image](https://user-images.githubusercontent.com/59611597/178808646-2d3ae27c-3b22-4868-b582-73e942b55b54.png)

**Reply by adityagoel28** [reply]
You can get your first Startstruck badge if your repo gets 16 stars.

**Reply by LuigiImVector** [reply]
You must have to do a commit with another person ([co-authoring commits](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors)) in a merged PR

**Reply by sibeux** [reply]
@LuigiImVector how we can add co-authors commit with github web?

**Reply by Daghall** [reply]
If you add a suggestion to a pull-request review, the author can accept your suggestion (committing it), and it will be co-authored by the you and the author.

If you amend/rebase a commit from someone else it also shows as co-authored, and I guess it would also trigger the achievement.

**Reply by luanribeiros** [reply]
very cool this!

**Reply by JhonnyLusonode** [reply]
Would you guys put a star on my repository so i can get the badge pls?⭐️
https://github.com/JhonnyLusonode/JhonnyLusonode
Thank you.

**Reply by ghost** [reply]
so good

**Reply by JhonnyLusonode** [reply]
Can someone help me out getting the Pair Extraordinaire badge pls?
I already made a co-author commit throught github desktop app but it didn´t work.. does someone has to do it and add me as a co-author or i should do the commit and co-auth someone?

**Reply by d-coder111** [reply]
> Can someone help me out getting the Pair Extraordinaire badge pls? I already made a co-author commit throught github desktop app but it didn´t work.. does someone has to do it and add me as a co-author or i should do the commit and co-auth someone?

Hey JhonnyLusonode,
If u wish u can add me as co-author and can commit to the repo👍😊

**Reply by JhonnyLusonode** [reply]
> > Can someone help me out getting the Pair Extraordinaire badge pls? I already made a co-author commit throught github desktop app but it didn´t work.. does someone has to do it and add me as a co-author or i should do the commit and co-auth someone?
> 
> Hey JhonnyLusonode, If u wish u can add me as co-author and can commit to the repo👍😊

Thank you!

**Reply by JhonnyLusonode** [reply]
> > Can someone help me out getting the Pair Extraordinaire badge pls? I already made a co-author commit throught github desktop app but it didn´t work.. does someone has to do it and add me as a co-author or i should do the commit and co-auth someone?
> 
> @JhonnyLusonode you have to add more than 2 co-authors to get this badge.
> 
> 1. create new PR
> 2. commit as more than 2 co-authors
> 3. merge.
>    If then, you will get this badge.

I´ve got the badge already!
Thank you! :D

**Reply by eliashz** [reply]
❤️

**Reply by mistletoeKANO** [reply]
I like this one.

**Reply by MarkusBansky** [reply]
This looks rally fun!

**Reply by AyatoKirishima** [reply]
This is so cute

**Comment by prathikshetty2002** [comment]
> Hi guys! I just got 2 badges from Github today. They are Starstruck and Quickdraw:
> 
> * Starstruck: appear when you create a repository and get many stars.
> * Quickdraw: appear when you create a new PR and they are lost within 5 minutes of opening
>   ![image](https://user-images.githubusercontent.com/71834167/178769787-0f60243f-1c2f-4aea-a886-eb001d264b55.png)
> 
> You can consult and admire them from my GitHub profile at [thuongtruong1009](https://github.com/thuongtruong1009) Have a nice day! 🥰

These are amazing i recieved the second one today. These are the base forms , they also have upgrades. If you want to know more about them see this repo : https://github.com/Schweinepriester/github-profile-achievements

**Reply by S2Sofficial** [reply]
Star ⭐ this Repo https://github.com/Swaroop2sky/swaroop2sky.github.io 

Thank you 😊

**Reply by VictorHugoAguilar** [reply]
I support you too, please support me. 

https://github.com/VictorHugoAguilar/ticketing-microservices-node-nextjs-react

thanks

**Reply by rohya8** [reply]
Star ⭐ this Repo https://github.com/rohya8/mean-stack-interview-questions
Thank you 😊

**Reply by elwynynion** [reply]
Star ⭐ this Repo https://github.com/elwynynion/ForDaFolio

**Reply by keithchad** [reply]
Please star this repo and i will star yours https://github.com/keithchad/design-toast

**Reply by keithchad** [reply]
> Star star this Repo https://github.com/elwynynion/ForDaFolio

Please star this repo and i will star yours https://github.com/keithchad/design-toast
I have already starred yours

**Reply by HarshitKumar9030** [reply]
Mine too https://github.com/HarshitKumar9030/HarshitKumar9030

**Comment by adityagoel28** [comment]
You can get your first Startstruck badge if your repo gets 16 stars.

**Reply by S2Sofficial** [reply]
Done, now please  ⭐ this Repo and help me to unlock this achievement.
https://github.com/Swaroop2sky/swaroop2sky.github.io

**Reply by number0x0001** [reply]
> Done, now please ⭐ this Repo and help me to unlock this achievement. https://github.com/Swaroop2sky/swaroop2sky.github.io

Done, please give mine a star: https://github.com/number0x0001/Seanux-Linux-Files-Scripts @S2Sofficial

**Reply by S2Sofficial** [reply]
> > Done, now please ⭐ this Repo and help me to unlock this achievement. https://github.com/Swaroop2sky/swaroop2sky.github.io
> 
> I have starred yours, star mine https://github.com/thenithinbalaji/YES-or-NO @S2Sofficial

Done bro, thank you

**Reply by S2Sofficial** [reply]
> > Done, now please ⭐ this Repo and help me to unlock this achievement. https://github.com/Swaroop2sky/swaroop2sky.github.io
> 
> Done, please give mine a star: https://github.com/number0x0001/Seanux-Linux-Files-Scripts @S2Sofficial

Done, and thank you

**Reply by S2Sofficial** [reply]
Sure, done, now your turn

**Reply by HarshitKumar9030** [reply]
Put a star, https://github.com/HarshitKumar9030/HarshitKumar9030

**Reply by Fudoshiki** [reply]
It doesn't work for repo created in organization...

**Comment by d-coder111** [comment]
> Hi guys! I just got 2 badges from Github today. They are Starstruck and Quickdraw:
> 
> * Starstruck: appear when you create a repository and get many stars.
> * Quickdraw: appear when you create a new PR and they are lost within 5 minutes of opening
>   ![image](https://user-images.githubusercontent.com/71834167/178769787-0f60243f-1c2f-4aea-a886-eb001d264b55.png)
> 
> You can consult and admire them from my GitHub profile at [thuongtruong1009](https://github.com/thuongtruong1009) Have a nice day! 🥰

Hey thuongtruong1009, Congrats for new achievements🎉👍

Hoping to connect with u on github to learn and collaborate together🤝😊

**Reply by NiisG** [reply]
I too got  2 new badges from Github today, YOLO and Starstruck
![image](https://user-images.githubusercontent.com/47427428/179051804-3ce24b7b-8253-4263-b54a-9cc5fe301701.png)

**Reply by s-sutharsan-20** [reply]
how to get these badges
who are eligible for these  badges please say

**Comment by fahimy143** [comment]


**Reply by fahimy143** [reply]
@flame4ost done

**Reply by d-coder111** [reply]
> Star this repo and comment your repo which i will star in return https://github.com/thenithinbalaji/YES-or-NO

✌👍

**Reply by fahimy143** [reply]
https://github.com/fahimy143/fahimy143

**Reply by number0x0001** [reply]
> https://github.com/fahimy143/fahimy143

@fahimy143 starred, please star in return: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by rohya8** [reply]
Star ⭐ this Repo https://github.com/rohya8/mean-stack-interview-questions
Thank you 😊

**Reply by thuongtruong1009** [reply]
> Hi all, for get starstruck need to reach 8 star. If u want u can put star on my readme repository, I will do the same for you. https://github.com/flame4ost/FingerprintApp

@flame4ost 
I starred this your repo. You also starred my repo at [here](https://github.com/thuongtruong1009/bee-music)

**Reply by d-coder111** [reply]
done✌👍 @thuongtruong1009
Check out my repo https://github.com/d-coder111/FNDetectoApp

Show some ❤ by starring it and I will appreciate it and will support u also😊🤝

**Comment by Mehul2203** [comment]
Hello everyone 👋, 
Congratulations on getting new achievements!! 🙌. 
I got one new "Quickdraw" achievement 💝. 
![achievements](https://user-images.githubusercontent.com/51510071/179120040-3a087dd7-4ad5-4601-8782-9fc8db55bdd1.PNG)
I really appreciate this new idea by GitHub of launching achievements to keep us motivated to work with team dynamics in an effective manner. 👍
Check out my profile @Mehul2203

**Comment by AlexRixten** [comment]


**Reply by VictorHugoAguilar** [reply]
I support you too, please support me.

https://github.com/VictorHugoAguilar/ticketing-microservices-node-nextjs-react

thanks

**Reply by number0x0001** [reply]
Please leave on here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Comment by melvincwng** [comment]
Sharing some of my repos as well, in case anyone of you guys are interested in starring ⭐:

1. [Facebook Clone](https://github.com/melvincwng/facebook-clone)
2. [Your First Open Source Project](https://github.com/Your-First-Open-Source-Project)
3. [Covid19SG](https://github.com/melvincwng/covid19-tracker-app)

Thanks all, happy to see such an inclusive community helping one another out 😊

**Reply by d-coder111** [reply]
> Sharing some of my repos as well, in case anyone of you guys are interested in starring ⭐:
> 
> 1. [Facebook Clone](https://github.com/melvincwng/facebook-clone)
> 2. [Your First Open Source Project](https://github.com/Your-First-Open-Source-Project)
> 3. [Covid19SG](https://github.com/melvincwng/covid19-tracker-app)
> 
> Thanks all, happy to see such an inclusive community helping one another out 😊

Starred [Facebook Clone](https://github.com/melvincwng/facebook-clone)😍⭐

**Reply by VictorHugoAguilar** [reply]
I support you too, please support me.

https://github.com/VictorHugoAguilar/ticketing-microservices-node-nextjs-react

thanks

**Reply by number0x0001** [reply]
Please leave on here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by melvincwng** [reply]
@prankshaw , nice project! Done 👍🏻

**Comment by cupid20103** [comment]
Please check my github badges!

**Reply by cyberbuddy-manas** [reply]
Looks awesome bro. Can you help me out to get the pair one if possible. I too want to have them on my profile.

**Reply by d-coder111** [reply]
> Please check my github badges!

Congratulations😊👍 and keep going🚀🎉

**Comment by cyberbuddy-manas** [comment]
Checkout my badges 🤩
![image](https://user-images.githubusercontent.com/70315395/179402971-0f190704-3f11-4be4-ae27-0b70cab8040b.png)

- [x] Quickdraw
- [x] Pull Shark
- [ ] Pair Extraordinaire
- [ ] Starstruck
- [ ] Yolo
- [ ] Galaxy Brain
- [ ] Public Sponsor

Will keep editing as I get more. Awesome feature for GitHub.

**Reply by d-coder111** [reply]
congrats cyberbuddy-manas 😊👍

**Comment by rohya8** [comment]
Star ⭐ this Repo(https://github.com/rohya8/mean-stack-interview-questions)
Thank you 😊

**Reply by d-coder111** [reply]
> Star ⭐ this Repo(https://github.com/rohya8/mean-stack-interview-questions) Thank you 😊

done✌👍
Check out my repo https://github.com/d-coder111/FNDetectoApp

Show some ❤ by starring it and I will appreciate it and will support u also😊🤝

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by Jon2G** [reply]
I leave a star, you leave a star https://github.com/Jon2G/Xamarin.Forms.UniformGrid 🤝

**Reply by ashishnpise** [reply]
I've given you star 

Hope i get one also

https://github.com/AshishisLiquid/arduinocar

**Reply by ashishnpise** [reply]
> I leave a star, you leave a star https://github.com/Jon2G/Xamarin.Forms.UniformGrid 🤝
I've given you star it would be great if you do also
https://github.com/AshishisLiquid/arduinocar

**Reply by d-coder111** [reply]
> @d-coder111 I put the last star for your repo. plz gimme also: https://github.com/dev-tom-0108/Lottery_DAO ;)

done @dev-tom-0108v👍😊

**Comment by itsmeshibintmz** [comment]
Beautiful badges.

**Comment by immortalmind2016** [comment]
![image](https://user-images.githubusercontent.com/22199342/179738970-5ab11035-dd5d-4687-ac98-2db952d95ca9.png)
 Hey Hey

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Comment by AlexRixten** [comment]


**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by keithchad** [reply]
Hello please star my  repo and I will star yours. https://github.com/keithchad/design-toast

**Reply by therohit777** [reply]
Recieved two amazing batches today

**Reply by d-coder111** [reply]
@therohit777 congrats 🎉😍 and good luck for future achievements 👍
Hoping to connect with u on github to learn together 😊🤝

**Comment by keithchad** [comment]
Hello. The badge seems intresting. Please star my repo and i will star yours so that we can all ge the badge. https://github.com/keithchad/design-toast

**Reply by michalkulisiewicz** [reply]
I did, if you don't mind returning the favor https://github.com/michalkulisiewicz/python-baselinker

**Comment by bogdanterzea** [comment]
I just got my `quickdraw` icon. That's cool!! 
![image](https://user-images.githubusercontent.com/49044962/180238838-34231a08-4927-45e8-8972-ac0bd1720f1d.png)

**Reply by michalkulisiewicz** [reply]
looks amazing, need to have it!

**Comment by NitinIsFit** [comment]
> When a repository gets many stars 

How many stars?

**Reply by bogdanterzea** [reply]
15

**Reply by NitinIsFit** [reply]
> 15

Thanks

**Comment by ShivaShirsath** [comment]


**Reply by michalkulisiewicz** [reply]
I did, if you don't mind returning the favor https://github.com/michalkulisiewicz/python-baselinker

**Comment by Wep-wep** [comment]
Pagaré! Gracias por ayudarme a undírme sin merecerlo. Dios bendiga todo..

**Comment by thuongtruong1009** [comment]
Oh, Thank you very much!
I also put a star on your repository!
Happy cooperate
________________________________
Từ: Sasuke ***@***.***>
Đã gửi: 04 Tháng Tám 2022 3:02 SA
Đến: community/community ***@***.***>
Cc: TRAN NGUYEN THUONG TRUONG ***@***.***>; Mention ***@***.***>
Chủ đề: Re: [community/community] New badges archivement appear! (Discussion #20443)


@d-coder111<https://github.com/d-coder111> I put the last star for your repo.
plz gimme also: https://github.com/dev-tom-0108/Lottery_DAO
;)

**Comment by vincode-io** [comment]
I got the quickdraw achievement accidentally by opening an issue to track it for release and then closing it almost immediately after a commit.

Maybe quickdraw shouldn't be given out if the person who closes the issue is the same as the person who opens it.

**Reply by billythekid** [reply]
I agree with you on this!

I'm incredibly annoyed that I don't yet have this quickdraw one yet. It's BY FAR the most important one to have, but I also refuse to game it for the badge! I'll just need to ~bribe~ encourage my team mates to fix those issues quicker!

🤠

**Comment by bloodybaker** [comment]
Hi, please, ⭐star⭐ my repo https://github.com/bloodybaker/PubgCheat
I'll be very grateful!

**Reply by JaimeTR** [reply]
Starred! Please star mine too  https://github.com/JaimeTR/instahack

**Comment by prathikshetty2002** [comment]
Hey guys i really want the starstruck badge pls star my repo : https://github.com/prathikshetty2002/Research-Nerd it would be gratious😊👍

**Reply by michalkulisiewicz** [reply]
I did, if you don't mind returning the favor https://github.com/michalkulisiewicz/python-baselinker

**Reply by prathikshetty2002** [reply]
Yes i too did 👍

**Reply by JaimeTR** [reply]
Starred! Please star mine too  https://github.com/JaimeTR/instahack

**Reply by prathikshetty2002** [reply]
> Starred! Please star mine too  https://github.com/JaimeTR/instahack

Yes done

**Reply by prathikshetty2002** [reply]
> Please star my repo
> I star back
> Https://github.com/ShivaShirsath/ShivaShirsath
> Thank you
> 

Yes i have starred your repo pls do mine also

**Comment by michalkulisiewicz** [comment]
Please, star⭐my repo https://github.com/michalkulisiewicz/python-baselinker looking for a work as a dev make them notice me!

**Reply by HarshitKumar9030** [reply]
Starred! Please star mine too https://github.com/HarshitKumar9030/HarshitKumar9030

**Comment by mistletoeKANO** [comment]


**Reply by keithchad** [reply]
I have started yours @mistletoeKANO please star mine https://github.com/keithchad/design-toast

**Reply by mistletoeKANO** [reply]
i too did

**Comment by mistletoeKANO** [comment]
Thank u

**Comment by Nielioo** [comment]
Please help me ⭐ this Repo, so I can unlock this achievement too. Thanks ^.^
https://github.com/Nielioo/Windrise-Genshin-Impact-Dictionary
https://github.com/Nielioo/Windrise-Genshin-Impact-Dictionary
https://github.com/Nielioo/Windrise-Genshin-Impact-Dictionary

![Starstruck](https://raw.githubusercontent.com/Nielioo/github-profile-achievements/main/images/starstruck-default.png)

**Reply by JaimeTR** [reply]
Starred! Please star mine too  https://github.com/JaimeTR/instahack

**Comment by mistletoeKANO** [comment]
I have unsubscirbed, thank u




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Hackers ***@***.***&gt;; 
发送时间: 2022年8月5日(星期五) 晚上8:17
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [community/community] New badges archivement appear! (Discussion #20443)





 
https://github.com/HackersTech/terminal-chat-ANON

**Comment by deprival** [comment]
Yo, can anyone star my repo?? https://github.com/vvent/404-Page

<3

**Reply by HarshitKumar9030** [reply]
Starred! Please star mine too https://github.com/HarshitKumar9030/HarshitKumar9030

**Reply by JaimeTR** [reply]
Starred! Please star mine too  https://github.com/JaimeTR/instahack

**Reply by deprival** [reply]
Thank you guys, sorry I forgot.

**Comment by Bendelk** [comment]
Thank you

**Comment by JaimeTR** [comment]
Hi, please, ⭐star⭐ my repo https://github.com/JaimeTR/instahack or https://github.com/JaimeTR/SvenBot

**Reply by HarshitKumar9030** [reply]
Starred! Please star mine too https://github.com/HarshitKumar9030/HarshitKumar9030

**Comment by HarshitKumar9030** [comment]
Hello, Please star my repo https://github.com/HarshitKumar9030/HarshitKumar9030

**Reply by JaimeTR** [reply]
ready my friend

**Comment by prathikshetty2002** [comment]
> plese star⭐ this repo [https://github.com/ShivaShirsath/ShivaShirsath](https://github.com/ShivaShirsath/ShivaShirsath)
> 
> Thank you 😇

Hi i have starred your repo can you star mine https://github.com/prathikshetty2002/Research-Nerd/

**Reply by deprival** [reply]
Could you star my repo? https://github.com/vvent/404-Page
(ill star yours)

**Reply by prathikshetty2002** [reply]
> Could you star my repo? https://github.com/vvent/404-Page
> (ill star yours)

Ok done pls do mine also

**Reply by deprival** [reply]
I did

**Comment by NitinIsFit** [comment]
Did you mean achievement instead of archivement?

**Comment by Zeioth** [comment]
What the fuck is this shit.

**Comment by deprival** [comment]
Yo, can anyone star my repo?? https://github.com/vvent/404-Page

<3

**Reply by Ufransa** [reply]
Done mate! could you star mine as well? [Repo](https://github.com/Ufransa/newsScrap)

**Reply by deprival** [reply]
starred :)

**Comment by drdreo** [comment]


**Reply by Ufransa** [reply]
Done mate! could you star mine as well? [Repo](https://github.com/Ufransa/newsScrap)

**Comment by 3kh0** [comment]
@queenofcorgis can you please lock this? people are starting to promote their repos. Thanks!

**Comment by Ufransa** [comment]
⭐⭐⭐ [STAR REPO THANKS !!!](https://github.com/Ufransa/newsScrap) ⭐⭐⭐

Hi all !!! Could you support my repo? Is the last proyect before I finish my career, still a work in progress and I though it would be a good chance to win this badge. I will star yours as well ;)

**Reply by thuongtruong1009** [reply]
Hi @Ufransa 
I starred your repo. Hope to collaborate happily
Thank you!
